tests/puppeteer/campaigns/tools/linkchecker.js

```diff
@@ -0,0 +1,129 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const urlsList = require('./urls.js');
+
+const LOG_PASSED = process.env.LOG_PASSED || false;
+
+
+// Report dir and file
+const reportPath = 'campaigns/tools/reports';
+let filename;
+
+const output = {
+  startDate: new Date().toISOString(),
+  endDate: null,
+  pages: [],
+};
+let outputEntry = {
+  name: '',
+  url: '',
+  passed: [],
+  failed: [],
+  jsError: [],
+};
+let requestError = false;
+let requestTextError = '';
+let javascriptError = false;
+let jsError = '';
```

Comment:
```suggestion
let javascriptTextError = '';
```
Maybe to stay coherent with `requestError` ?

Reply:
done